lib/Doctrine/ODM/PHPCR/Decorator/DocumentManagerDecorator.php

```diff
@@ -235,7 +235,7 @@ public function getDocumentsByPhpcrQuery(QueryInterface $query, $className = nul
      */
     public function persist($document)
     {
-        return $this->wrapped->persist($document);
+        $this->wrapped->persist($document);
```

Comment:
declared to return `void`, lets not trick people into thinking this could return something.